airflow/models/baseoperator.py

```diff
@@ -1141,7 +1178,7 @@ def set_upstream(self, task_or_task_list: Union['BaseOperator', List['BaseOperat
 
     @property
     def output(self):
```

Comment:
What about other xcom values than the default `return_value` key?

Reply:
As far as I know you can use `op.output["your key"]` because `XComArg` implements `__getitem__`.